index.rst

```diff
@@ -2,7 +2,19 @@ Raiden Network Specification
 =================================
 
 
-This is a tentative specification for the `Raiden Network <http://raiden.network/>`_. It's under development and will be further refined in subsequent iterations.
+This is the specification of the `Raiden Network <http://raiden.network/>`_.
+
+Its goal is to provide an exact description of the Raiden protocol. So the structure of
+a mediated transfer, message formats and such are to be found here, while implementation
+specific parts of the Raiden client, which is the protocol's reference implementation, are
+documented in the `docs <https://raiden-network.readthedocs.io/en/stable/index.html>`
+instead.
+
+Smart contracts and services, however, are specified in greater detail here since every possible
+implementation of Raiden needs to be developed against them. This makes it necessary, for example,
+to specify the API endpoints for the services, whereas the API of the Raiden client is not documented
+here but in the Raiden docs.
+
```

Comment:
`This makes it necessary, for example,
to specify the API endpoints for the services, whereas the API of the Raiden client is not documented
here but in the Raiden docs.` 

Sounds like a justification, please remove.